cc: @mstabc 
Two unit tests failed. These are the logs:
```bash
=========================== short test summary info ============================
FAILED packages/railtracks/tests/unit_tests/guardrails/test_block_text_input_guard.py::TestBlock::test_block_reason_contains_pattern - AssertionError: assert 'bad_word' in 'Input blocked: prohibited content detected.'
 +  where 'Input blocked: prohibited content detected.' = GuardrailDecision(action=<GuardrailAction.BLOCK: 'block'>, reason='Input blocked: prohibited content detected.', messages=None, output_message=None, user_facing_message=None, meta=None).reason
FAILED packages/railtracks/tests/unit_tests/guardrails/test_block_text_output_guard.py::TestBlock::test_block_reason_contains_pattern - AssertionError: assert 'SECRET' in 'Output blocked: prohibited content detected.'
 +  where 'Output blocked: prohibited content detected.' = GuardrailDecision(action=<GuardrailAction.BLOCK: 'block'>, reason='Output blocked: prohibited content detected.', messages=None, output_message=None, user_facing_message=None, meta=None).reason
 ```

--- a/packages/railtracks/tests/unit_tests/guardrails/test_block_text_input_guard.py
+++ b/packages/railtracks/tests/unit_tests/guardrails/test_block_text_input_guard.py
@@ -48,11 +48,12 @@ class TestBlock:
         decision = guard(event)
         assert decision.action == GuardrailAction.BLOCK
 
-    def test_block_reason_contains_pattern(self) -> None:
+    def test_block_reason_does_not_leak_pattern(self) -> None:
         guard = BlockTextInputGuard(pattern=r"bad_word")
         event = _make_input_event(MessageHistory([UserMessage("This has bad_word")]))
         decision = guard(event)
-        assert "bad_word" in decision.reason
+        assert decision.reason == "Input blocked: prohibited content detected."
+        assert "bad_word" not in decision.reason
 
 
 class TestAllow:

--- a/packages/railtracks/tests/unit_tests/guardrails/test_block_text_output_guard.py
+++ b/packages/railtracks/tests/unit_tests/guardrails/test_block_text_output_guard.py
@@ -28,11 +28,12 @@ class TestBlock:
         decision = guard(event)
         assert decision.action == GuardrailAction.BLOCK
 
-    def test_block_reason_contains_pattern(self) -> None:
+    def test_block_reason_does_not_leak_pattern(self) -> None:
         guard = BlockTextOutputGuard(pattern=r"SECRET")
         event = _make_output_event("The SECRET is here")
         decision = guard(event)
-        assert "SECRET" in decision.reason
+        assert decision.reason == "Output blocked: prohibited content detected."
+        assert "SECRET" not in decision.reason
 
 
 class TestAllow:

